sensor_msgs/Image request
#Matrix4x4 robotToCameraMatrix #Get current camera position in NAO space
---
bool isQRcodeFound
uint32 numberOfQRcodes
Matrix4x4 cameraToQRcode
Matrix4x4 robotToQRcode
string[] message
#float32[] cornersMatrix
